docs/open_source/scan/scan_vision/index.md

```diff
@@ -17,68 +15,339 @@ pip install giskard giskard-vision
+
+> ### ⚠️ Warning
+>
+> It's highly recommended that you wrap your data **before preprocessing** so that you can easily interpret
+> the scan results.
+>
+> For now, the image classification scan only supports one label per image
+>
+
+```python
+from giskard_vision.core.dataloaders.base import DataIteratorBase
+from giskard_vision.core.dataloaders.meta import MetaData
+from giskard_vision.core.issues import EthicalIssueMeta, PerformanceIssueMeta
+
+
+class DataLoaderClassification(DataIteratorBase):
+
+    @property
+    def idx_sampler(self) -> np.ndarray:
+        """Accesses the array of element indices to iterate over
+
+        Returns:
+            np.ndarray: The array of indices
+        """
+        return list(range(len(self.image_paths)))
+
+    @classmethod
+    def get_image(self, idx: int) -> np.ndarray:
+        """
+        Returns a single image from an index as numpy array
+
+        Args:
+            idx (int): Index of the image
+
+        Returns:
+            np.ndarray: The image as numpy array (h, w, c)
+        """
+        return self.images_paths[idx]
+
+    @classmethod
+    def get_label(self, idx: int) -> Optional[np.ndarray]:
+        """
+        Gets the label (for a single image) for a specific index.
+
+        Args:
+            idx (int): Index of the image.
+
+        Returns:
+            Optional[str]: Label for the given index.
+        """
+        return 'label'
+    
+    @classmethod
+    def get_meta(self, idx: int) -> Optional[MetaData]:
+        """
+        Gets meta information (for a single image) for a specific index.
+
+        Args:
+            idx (int): Index of the image.
+
+        Returns:
+            Optional[MetaData]: Meta information for the given index.
+        """
+        default_meta = super().get_meta() # To load default metadata
+        return MetaData(
+            data={
+                **default_meta.data,
+                'meta1': 'value1',
+                'meta2': 'value2',
+                'categorical_meta1': 'cat_value1',
+                'categorical_meta2': 'cat_value2'
+            },
+            categories=default_meta.categories+['categorical_meta1', 'categorical_meta2'],
+            issue_groups={
+                **default_meta.issue_groups,
+                'meta1': PerformanceIssueMeta,
+                'meta2': EthicalIssueMeta,
+                'categorical_meta1': PerformanceIssueMeta,
+                'categorical_meta2': EthicalIssueMeta,
+            }
+        )
+
+
+giskard_dataset = DataLoaderClassification()
+```
+::::::
+::::::{tab-item} Object detection
+
+To scan your model, start by **wrapping your dataset** with `DataIteratorBase`. Your class should override:
+
+- `idx_sampler` (property) that returns the array of indices to iterate over
+- `get_image` that loads the image corresponding to the right index as a `np.ndarray`
```

Comment:
to be adjusted: a dict containing `boxes` (np.array) and `labels` (string) as keys